aptos-move/framework/aptos-framework/sources/BigVector.move

```diff
@@ -14,7 +19,8 @@ module AptosFramework::BigVector {
     /// A Scalable vector implementation based on tables, elements are grouped into buckets with `bucket_size`.
     struct BigVector<T> has store {
         buckets: Table<u64, vector<T>>,
-        next_index: BigVectorIndex,
+        length_index: BigVectorIndex,
+        bucket_capacity: u64,
```

Comment:
nit: I'd call it `capacity_in_buckets`, `num_buckets`, `num_buckets_reserved`, etc. -- buckets_capacity sounds like the size of a bucket.